src/components/3d-emoji-picker/ThreeDEmojiPicker.tsx

```
@@ -50,14 +50,13 @@ export const ThreeDEmojiPicker = ({ onEmojiSelect }: Props) => {
             >
               <div className="flex flex-wrap justify-center gap-1">
                 {Object.values(emojis).map((emoji) => (
-                  <button
+                  <div
                     key={emoji.cldr}
-                    type="button"
                     onClick={() => handleEmojiClick(emoji)}
                     className={emoji.cldr}
                   >
```

Biome

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)